src/api/config/environments/production.rb

```diff
@@ -71,6 +71,18 @@
     config.logger = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  # Use lograge to show the logs in one line
+  config.lograge.enabled = true
+  config.lograge.custom_options = lambda do |event|
+    exceptions = ['controller', 'action', 'format', 'id']
+    {
+      params: event.payload[:params].except(*exceptions),
```

Comment:
@mdeniz this that you posted in the description:

```
method=POST path=/comments format=js controller=Webui::CommentsController action=create status=200 duration=1436.57 view=77.25 db=13.92 params={"utf8"=>"✓", "commentable_type"=>"Project", "commentable_id"=>"1", "comment"=>{"body"=>"hola que tal"}, "commit"=>"Add comment"} host=127.0.0.1 time=2018-05-14 15:59:03 +0000 user=Admin
```
is the result of this code? because I think you are excluding here controller and action and they are in the example :thinking:

Reply:
Is a C&P from my local server with that config. controller and action are fourth and fifth values, those come as default from the gem... :open_mouth:

Reply:
true :+1: